json_serializable/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 0.5.5
+## 0.5.4+1
 
 * Added named `formatOutput` parameter to `jsonPartBuilder`.
```

Comment:
adding a parameter (A feature someone can depend on) should be done in a minor release, not a bug fix release. The previous version in this file was better.

Reply:
Er, uh. Yup. Will fix.

...although the named arg will be useful to very few people. But fair...

Reply:
Went the other way and removed the changelog entry – jsonPartBuilder is not public!